docs/reference/snapshot-restore/apis/get-snapshot-api.asciidoc

```diff
@@ -158,11 +163,11 @@ created by an SLM policy but not those snapshots that were not created by an SLM
 policy you can use the special pattern `_none` that will match all snapshots without an SLM policy.
 
 NOTE: The `after` parameter and `next` field allow for iterating through snapshots with some consistency guarantees regarding concurrent
-creation or deletion of snapshots. It is guaranteed that  any snapshot that exists at the beginning of the iteration and not concurrently
+creation or deletion of snapshots. It is guaranteed that any snapshot that exists at the beginning of the iteration and is not concurrently
 deleted will be seen during the iteration. Snapshots concurrently created may be seen during an iteration.
 
-NOTE: The parameters `size`, `order`, `after`, `offset`, `slm_policy_filter` and `sort` are not supported when using `verbose=false` and
-the sort order for requests with `verbose=false` is undefined.
+NOTE: The parameters `size`, `order`, `after`, `after_value`, `offset`, `slm_policy_filter` and `sort` are not supported when using
```

Comment:
We should also note the incompatibility of `after_value` with `offset` and `after`?

Reply:
Well now that you pointed it out below, it would only be with "after" in the final thing which I'd code up very shortly after this one. Not sure it's worth the added text though, it obviously makes no sense setting both, the request validation will tell you so and we already point this out in the `after` section?

Reply:
I see you added the incompatibility note to `offset`, I missed that on my initial read. I think it still makes sense to add to either `after` or `after_value` that they are incompatible. I could see this being slightly surprising, since `after_value` is like a filter so it could be surprising that it does not work with pagination using `after` (and this could slip through some UI testing).

Reply:
Done, added a note to `after` :)